pyproject.toml

```
@@ -9,7 +9,7 @@ readme = "README.rst"
 requires-python = ">=3.8"
 license = {text = "BSD License"}
 authors = [{name = "The PyGMT Developers", email = "pygmt.team@gmail.com"}]
-keywords = []
+keywords = ["geospatial", "cartography"]
```

Add two more keywords. Maybe we first simply collect some keywords, and determine the final order in the end 🙂?
```suggestion
keywords = ["geospatial", "cartography, geophysics, seismology"]
```